api/src/main/proto/stellarstation/api/v1/radio/radio.proto

```diff
@@ -43,6 +43,9 @@ message RadioDeviceConfiguration {
   // The protocol used by this device when doing RF communication. If unset, the device is only
   // demodulating / modulating without applying any higher-level communication protocol.
   CommunicationProtocol protocol = 4;
+
+  // The polarization used by this radio device.
+  Polarization polarization = 5;
```

Comment:
Which does this mean,  a polarization used in a communication or polarizations supported by the radio?

Reply:
`RadioDeviceConfiguration` is a configuration that will be used for actual communication.

Reply:
Oops finally remembered why I'm feeeling dejavu

We have a polarization enum

https://github.com/infostellarinc/stellarstation-api/blob/75e7945ee70becfa692cce891b96bd7ad9193d20/api/src/main/proto/stellarstation/api/v1/antenna/antenna.proto#L48

And we return monitoring information in `AntennaState`

https://github.com/infostellarinc/stellarstation-api/blob/50d93d713cdab00892ec980e7f855998c90e954c/api/src/main/proto/stellarstation/api/v1/monitoring/monitoring.proto#L214

My understanding is polarization is a feature of the antenna, not the radio which is why it's there (we don't get the monitoring data for LHCP vs RHCP from a radio device but from an ACU). I guess polarization should be next to TLE in our protos.